<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelTile</class>
 <widget class="QWidget" name="ChannelTile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>162</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>162</width>
    <height>162</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	color: rgba(255,255,255,190);
}

#updated {
	font: 20px;
	font: bold;
}

QPushButton {
    border: none;
}
#openButton:!hover {
    background-color: rgba( 0,0,0, 100);
}
#openButton:hover:!pressed {
    background-color:   rgba( 0, 0, 0, 120 );
}
#openButton:pressed {
    background-color:   rgba( 0, 0, 0, 140);
}

#infoOverlay {
	background-color: rgba(0,0,0,120);
}

#title {
	font: 16pt
}

#titleSep {
	color: rgba(255,255,255,100)
}

#uploaded {
	font: 16px;
}

#ratingOverlay {
	background-color: rgba(0,0,0,70);
}
#uploadCount {
	color: rgb(50,255,50);
}

#subscriberCount {
	color: rgb(255,255,30);
}

#viewCount {
	color: rgb(30, 120, 255);
}


</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>162</height>
    </rect>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>162</width>
     <height>162</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <widget class="QWidget" name="overlaylayout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>162</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="infoOverlay" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="title">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Title</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="updated">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>139</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>updated</string>
    </property>
   </widget>
   <widget class="QWidget" name="ratingOverlay" native="true">
    <property name="geometry">
     <rect>
      <x>121</x>
      <y>112</y>
      <width>41</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="viewCount">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="uploadCount">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="subscriberCount">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
